src/components/dashboard/TransactionConfirmation.js

```diff
@@ -46,6 +46,10 @@ const TransactionConfirmation = ({ screenProps, styles }: ReceiveProps) => {
 
   const handlePressDone = useCallback(() => goToRoot(), [goToRoot])
 
+  const secondTextPoint = useMemo(
+    () => (action === ACTION_SEND ? 'Share it with your recipient' : 'Share it with sender'),
```

Comment:
in this case useMemo isn't required as
1. wrapped function doesn't performs operatings tasking a lot of CPU time and doesn't uses/returns large data amounts
2. it's result is an string primitive, not an object-like variable. It's compared by its value while passed to component props (object-like props or callbacks are comapred by the reference so that why we wrap them onto useMemo/Callback)

Reply:
done